qiskit_optimization/problems/quadratic_program.py

```diff
@@ -917,45 +916,47 @@ def export_as_lp_string(self) -> str:
         Returns:
             A string representing the quadratic program.
         """
+        warn(
```

Comment:
There are some deprecation decorators  in Qiskit that we try and use now. They also emit the deprecation into the docstring so the notices come out in the published docs. You can most likely find usage examples in Qiskit - this is the file with the decorators https://github.com/Qiskit/qiskit/blob/main/qiskit/utils/deprecation.py

Reply:
Thanks again for the reference - used the appropriate decorators in the last commit.